package com.rh.core.util.exception;


public class ExceptionUtilTest {

	/*@Test
	public void testAbc() {
		try {
			
            throw new RuntimeException("fuck");
        } catch (Exception e) {
            System.out.println("-----------------------");
            System.out.println(ExceptionUtil.toMsgString(e));
            System.out.println("-----------------------");
        }
	}*/

}
